plutus-core/plutus-core/src/PlutusCore/Default/Builtins.hs

```diff
@@ -786,6 +791,7 @@ instance Flat DefaultFun where
               Sha3_256                 -> 19
               Blake2b_256              -> 20
               VerifySignature          -> 21
+              VerifySECP256k1Signature -> 51
```

Comment:
In fact, maybe that should go at the end.  If someone else wants to add a new builtin they might look at the end of the list and think that 51's the next free tag.

Reply:
So do you suggest I use the largest number available for `VerifySECP256k1Signature` instead of `51`? There's no specified system of numbering these in the documentation of the module.

Reply:
No, I meant that if you look at the end of that list of cases it says
```
              UnListData               -> 44
              UnIData                  -> 45
              UnBData                  -> 46
              EqualsData               -> 47
              MkPairData               -> 48
              MkNilData                -> 49
              MkNilPairData            -> 50
```
so it would be easy to think that the next free number is 51.  Of course if we arrange them numerically then it moves `VerifySECP256k1Signature` away from the rest of the cryptographic builtins, so you get a different kind of confusion.